scripts/migration.sh

```diff
@@ -135,12 +136,12 @@ do
     migrate-l10n \
         --lang ${locale} \
         --reference-dir ${quarantine_path} \
-        --localization-dir ${l10n_clones_path}/${locale} \
+        --localization-dir ${l10n_path}/${locale} \
         ${dry} ${recipes_list}
-
-    # Push to hg server
-    if [ "${push_repository}" = true ]
-    then
-        hg --cwd ${l10n_clones_path}/${locale} push
-    fi
 done
+
+# Push git repository
+if [ "${push_repository}" = true ]
+then
+    git -C ${l10n_path} push
```

Comment:
Should we be more specific instead of just doing `git push`? Potentially someone's local config could have `push.default` set to something different. (This would also apply to other `git push` commands / subprocess.run commands as well.)

Either that or we should point it out what the expected config for `push.default` is in the Readme (I'm assuming `simple`).

Reply:
Are you asking to specify the remote, the branch, or both?

Given it's only two people using this, I think it's safe to assume that we either have the default value (both `matching` and `simple` would work, since we work on the default branch).

The side question is we should use a branch and a pull request for migration…